graphics/rendering/blender/blender.def

```diff
@@ -1,40 +1,39 @@
-Bootstrap: docker
+Bootstrap: library
 From: ubuntu:16.04
 
 %post
-    apt-get update
-    apt-get -y install blender blender-data
+apt-get update
+apt-get -y install blender blender-data
 
 %help
-    singularity run blender.sif [scene file] [output directory] <frame | start frame:end frame>
+singularity run blender.sif [scene file] [output directory] <frame | start frame:end frame>
 
-    Example:
-    Using `my_scene.blend`, render all frames, and output into `run/output`
+Example:
+Using `my_scene.blend`, render all frames, and output into `run/output`
 
-        singularity run blender.sif my_scene.blend run/output
+    singularity run blender.sif my_scene.blend run/output
```

Comment:
Same here, just for context add `$` before the command, check and add this for all the help section commands

Reply:
:+1: